consul/config_templates/consul-values-1.9.1-demo.yaml

```diff
@@ -0,0 +1,1841 @@
+  # The default setting, i.e. serviceaccount.name!=default, prevents the
+  # 'default' Service Account from logging in.
+  # If set to an empty string all service accounts can log in.
+  # This only has effect if ACLs are enabled.
+  #
+  # See https://www.consul.io/docs/acl/acl-auth-methods.html#binding-rules
+  # and https://www.consul.io/docs/acl/auth-methods/kubernetes.html#trusted-identity-attributes
+  # for more details.
+  # Requires Consul >= v1.5 and consul-k8s >= v0.8.0.
+  aclBindingRuleSelector: "serviceaccount.name!=default"
+
+  # If you are not using global.acls.manageSystemACLs and instead manually setting up an
+  # auth method for Connect inject, set this to the name of your auth method.
+  overrideAuthMethodName: ""
+
+  # Refers to a Kubernetes secret that you have created that contains
+  # an ACL token for your Consul cluster which allows the Connect injector the correct
+  # permissions. This is only needed if Consul namespaces [Enterprise Only] and ACLs
+  # are enabled on the Consul cluster and you are not setting
+  # `global.acls.manageSystemACLs` to `true`.
+  # This token needs to have `operator = "write"` privileges to be able to
+  # create Consul namespaces.
+  aclInjectToken:
+    secretName: null
+    secretKey: null
+
+  # Requires Consul >= v1.5 and consul-k8s >= v0.8.1.
+  centralConfig:
+    # Controls whether central config is enabled on all servers and clients.
+    # See https://www.consul.io/docs/agent/options.html#enable_central_service_config.
+    # If changing this after installation, servers and clients must be restarted
+    # for the change to take effect.
+    enabled: true
+
+    # Allows you to specify a convenience default protocol if
+    # most of your services are of the same protocol type. The individual annotation
+    # on any given pod will override this value.
+    # Valid values are "http", "http2", "grpc" and "tcp".
+    # @type: string
+    defaultProtocol: null
+
+    # Raw json string that will be written as the value of
+    # the "config" key of the global proxy-defaults config entry.
+    # See: https://www.consul.io/docs/agent/config-entries/proxy-defaults.html
+    # NOTE: Changes to this value after the chart is first installed have _no_
+    # effect. In order to change the proxy-defaults config after installation,
+    # you must use the Consul API.
+    proxyDefaults: |
+      {}
+
+  sidecarProxy:
+    # Set default resources for sidecar proxy. If null, that resource won't
+    # be set.
+    # These settings can be overridden on a per-pod basis via these annotations:
+    #
+    # - `consul.hashicorp.com/sidecar-proxy-cpu-limit`
+    # - `consul.hashicorp.com/sidecar-proxy-cpu-request`
+    # - `consul.hashicorp.com/sidecar-proxy-memory-limit`
+    # - `consul.hashicorp.com/sidecar-proxy-memory-request`
+    resources:
+      requests:
+        # Recommended default: 100Mi
+        # @type: string
+        memory: null
```

Comment:
If we did not set the resource limit, Is there will the sidecar make a broken of the pod with OOM or others?  I mean although `sidecarProxy` needs to keep work well as long as it can be. But if it accounts more resource and trigger the pod limit and let the pod broken

Reply:
Yes, I can set the recommended values.

Reply:
Thanks